tool-plugins/intellij/README.md

```diff
@@ -1,57 +1,89 @@
+    - [Find Usage](getting-started/plugin-features/README.md#find-usage)
+    - [Go to Definition](getting-started/plugin-features/README.md#go-to-definition)
+    - [Formatting](getting-started/plugin-features/README.md#formatting)
+    - [Parameter Information](getting-started/plugin-features/README.md#parameter-information)
+    - [Quick Documentation](getting-started/plugin-features/README.md#quick-documentation)
+    - [Object/Record Fields suggestions](getting-started/plugin-features/README.md#objectrecord-fields-suggestions)
+    - [Annotation fields suggestions](getting-started/plugin-features/README.md#annotation-fields-suggestions)
+    - [File Templates](getting-started/plugin-features/README.md#file-templates)
+    - [Live Templates and Code snippets](getting-started/plugin-features/README.md#live-templates-and-code-snippets)
+    - [Spell Checking](getting-started/plugin-features/README.md#spell-checking)
+    - [Semantic Analyzing](getting-started/plugin-features/README.md#semantic-analyzing) 
+
+- [**Plugin Developer Guide**](getting-started/plugin-developer-guide/README.md#plugin-developer-guide)
+    - [Testing/Debugging the plugin using IntelliJ](getting-started/plugin-developer-guide/README.md#testingdebugging-the-plugin-using-intellij-idea)
+    
+- [**Release versions schema**](#release-versions-schema)
+- [**How to Contribute**](#how-to-contribute) 
+- [**Licence**](#license)
+- [**Copyright**](#copyright)
+- [**Useful Links**](#useful-links)
+
+
+## How to Install
 
 ### From Jetbrains plugin repository
 1. Go to **Settings** (**Preferences** in **MacOS**)-> **Plugins**. 
 2. Select **Browse Repositories** button at the bottom. 
 3. Search for **Ballerina** using the search box. Ballerina plugin will show up. Then you can install the plugin using the Install button.
 4. Restart IDEA.
 
-**Note:** Refer the [Getting Started](https://github.com/ballerinalang/plugin-intellij#getting-started) section to view details about setting up Ballerina SDK.
-
 ### From a local build
-1. Go to **File -> Settings** (**IntelliJ IDEA -> Preferences** in **macOS**) and select **Plugins**.
-2. Click **Install plugin from disc** button and select the deployed **plugin zip** file. Please make sure to install the Zip file, not the extracted Jar files. This zip contains an additional library as well. Without this library, the plugin will not work properly.
-3. Restart IDEA.
-
-## Release versions schema
-
-Below you can see the versions of the plugin which correspond to the versions of the 
-IntelliJ Platfom.
-
-| Plugin Version | Platform Version |
-| --- | --- |
-| 0.8.0 - 0.8.2 | IntelliJ IDEA 2016.3+ (Does not support IntelliJ IDEA 2017+)|
-| 0.8.3 - 0.981.0 | IntelliJ IDEA 2016.3+ (Supports IntelliJ IDEA 2017+)|
-| 0.982.0+ | IntelliJ IDEA 2017.3+|
-
-## Getting started
+1. Run `./gradlew buildPlugin`. In the **build/distributions** directory, **ballerina-intellij-idea-plugin-[VERSION].zip** will be created.
```

Comment:
Change to "1. Run `./gradlew buildPlugin`. **ballerina-intellij-idea-plugin-[VERSION].zip** is created in the **build/distributions** directory."